source/docs/yearly-overview/known-issues.rst

```diff
@@ -118,3 +118,10 @@ Issues with WPILib Dashboards and Simulation on Windows N Editions
 - Robot Simulation will crash on start-up
 
 **Solution:** Install the `Media Feature Pack <https://www.microsoft.com/en-us/software-download/mediafeaturepack>`__
+
+SysId - Robot program crash on startup when using CAN Spark Maxes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Issue:** SysId 2023.1.1's deployed robot program crashes on startup if it was configured to use CAN Spark Maxes.
+
+**Workaround:** Download a fixed version for your OS from `here <https://github.com/wpilibsuite/sysid/actions>`__ (see the "Summary" section of the "Build" job).
```

Comment:
This link is really vague and teams can easily get lost there. We should either link to filter so only "Build" jobs are displayed (ie https://github.com/wpilibsuite/sysid/actions/workflows/build.yml) or pin to a specific version.

Reply:
I didn't link to a specific set of artifacts because they expire after 3 months. Adding a filter makes sense though.